### DISCUSSION TITLE
Swipe title bar to direction action

### DISCUSSION THREAD

**Discussion by epitonium** [rxhanson/Rectangle#980] [Ideas]
I use similar app "Grid" (https://macgrid.app) for managing macOS windows and there is a "Swipe" feature for trackpad users which can be used to place my windows Left_Half/Right_Half/Bottom_Half/Expand by simply swiping the window's title bar Left/Right/Bottom/Top with two fingers without even touching the keyboard.

**Comment by rxhanson** [comment]
I don't have plans to add this into Rectangle, but see no reason to turn down a good pull request for it. At first cut, I would prefer this to be added in as a terminal command config only, not in the UI.


As an aside, for trackpad users, I strongly prefer using the "rest & remove all but one" gesture for window throw that is in [Rectangle Pro](https://rectangleapp.com/pro). The benefits of the Rectangle Pro implementation include:

- You can use it with the cursor anywhere on the window, not just in the title bar, which can make it much faster to get windows where you want.
- The window throw with this gesture has up to 16 different window actions available, and all of it is customizable.
- I've found that I'm less likely to accidentally trigger this gesture than scrolling on a title bar.

Of course, it's not as simple as a swipe and has a little learning curve to it.
